app/javascript/components/modals/contact-us/styles.scss

```diff
@@ -1,85 +1,93 @@
 @import '~styles/settings.scss';
 $mobile-padding: rem(20px);
 $desktop-padding: rem(40px);
+$mobile-margin: rem(8px);
```

Comment:
I wonder if Fausto meant it to be 10px (like all other mobile margins). Often we round things like this unless the designer has specified this to be exact.